certified-connectors/Dokobit Portal/apiDefinition.swagger.json

```diff
@@ -1462,6 +1601,401 @@
+              "type": "object",
+              "required": ["file"],
+              "properties": {
+                "file": {
+                  "type": "object",
+                  "required": ["name", "content"],
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "File name.",
+                      "x-ms-summary title": "Name",
+                      "x-ms-visibility": "important"
+                    },
+                    "content": {
+                      "type": "string",
+                      "description": "Base64 encoded file content."
+                      }
+                    }
+                  }
+                }
+              },
+            "x-ms-visibility": "important"
+            }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Upload status"
+                },
+                "token": {
+                  "type": "string",
+                  "description": "Validation token",
+                  "title": "token"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "400",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
+                        "type": "string",
+                        "description": "message"
+                      },
+                      "error_code": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "error_code"
+                      },
+                      "field": {
+                        "type": "string",
+                        "description": "field"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                },
+                "error_code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "error_code"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/validation/{token}/download/detailed-report-pdf": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@epelanis please give a proper description for your connector responses. for example you can use "200", "OK", "Sucess", "Bad Request", etc.
Please fix all the responses in the swagger file.

Reply:
@Amjed-Ayoub Could you suggest a description in cases where we return a file?

Reply:
@Amjed-Ayoub Fixed